README.md

```diff
@@ -604,8 +602,6 @@ It's based on the `vendor/bin/behat -di @cms` output.
 
 	 Then /^I should see the CMS$/
 
-	 Then /^I should see a "([^"]*)" notice$/
```

Comment:
Why is this being removed?

Reply:
I removed this pattern from `silverstripe-framework`. [Please, have a look here](https://github.com/silverstripe/silverstripe-framework/pull/10594/files#diff-b5ac2578edc2e46f01f689ceab76303bb7c8897d79e80962a1d6d65e18db7595L80-L86)